docker/cluster_test/run.sh

```diff
@@ -2,9 +2,13 @@
 export ALLOW_DEPLOY=yes
 unset AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN
 
-mv run.out run.out.bak || echo 'run.out does not exist'
-if ! ./cluster-test --workplace cluster-test --run > run.out ; then
-  REQUEST=\{\"text\":\"cluster_test\ terminated\"\}
+
+while true; do
+  mv run.out run.out.bak || echo 'run.out does not exist'
```

Comment:
Not sure if you'd want to save and tag the output of the failed test for later debugging.  Something to consider.

Reply:
Probably something we can do later. For now we have at least the output of the last failed run.